extensions/lifecycle/LifecycleConfigValidator.js

```diff
@@ -67,6 +67,13 @@ const joiSchema = joi.object({
     coldStorageRestoreAdjustTopicPrefix: joi.string().default('cold-restore-adjust-req-'),
     coldStorageGCTopicPrefix: joi.string().default('cold-gc-req-'),
     coldStorageStatusTopicPrefix: joi.string().default('cold-status-'),
+    supportedLifecycleRules: joi.array().items(joi.string()).default([
```

Comment:
we may also want to validate each item from this list : i.e. to ensure each value is indeed of the potentially supported rule type

Reply:
done here : https://github.com/scality/backbeat/pull/2548/commits/1d605348937a9adcde59b729c1708b49d714167c#diff-4ddf07d1cd40d3d68a85bcf7ab41f36f43094af8cfc0e93424067c48bd03b623R70-R78